### DISCUSSION TITLE
Student benefits

### DISCUSSION THREAD

**Discussion by floppek** [community/community#139268] [GitHub Education] [answered]
### Hi everyone,

hey been waiting for my pack some time and since im very unpatient any eta for when will i get it?
![Zrzut ekranu 2024-09-21 104907](https://github.com/user-attachments/assets/608f6fa8-1f21-473c-a6a8-73c01ad1de4d)

**Comment by davevad93** [comment, marked as answer]
Hi @floppek ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @floppek was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by Djarindo** [comment]
![image](https://github.com/user-attachments/assets/c89e3367-9e26-4941-b709-4ab7c09f01a7)
I'm in same situation now(

**Reply by devartes** [reply]
![Captura de Tela 2024-09-21 às 07 36 30](https://github.com/user-attachments/assets/cb2cf249-5067-4bc9-be48-5d136c248126)
I'm in this situation too, I believe we need to wait a little longer to have access.

**Reply by floppek** [reply]
i guess we just have to wait 🤷 i will update you guys if i get my benefits

**Reply by devartes** [reply]
I received my access today 😄

**Reply by manhcuongkm** [reply]
![1](https://github.com/user-attachments/assets/894c2b16-b9ed-451d-ab44-cacff51d528b)
I'm on the same day as you, but mine still hasn't been granted access.

**Reply by 4oodo** [reply]
me too 
![image](https://github.com/user-attachments/assets/c57b3dc2-9411-4533-9e4c-55c4e1332359)

**Reply by floppek** [reply]
> I received my access today 😄

i ddint :(

**Reply by floppek** [reply]
![image](https://github.com/user-attachments/assets/42b61afc-0722-4f1b-a1f1-e9894246ecf6)
got mine like 1h ago:)

**Reply by Djarindo** [reply]
Congrats! But I still waiting my moment

**Reply by 4oodo** [reply]
lets go 
![image](https://github.com/user-attachments/assets/146c2bf6-9374-45e1-9669-bd8c990d2827)

**Comment by rakibnsajib** [comment]
I usually takes around 2-3 days and till 5days long. Once your application is approved, you will get the benefits also.

> Congratulations!

**Comment by Djarindo** [comment]
![image](https://github.com/user-attachments/assets/41f21847-071b-47cf-948f-ec6ff562c117)
Oh yeah,I received my benefits exactly after 3 days of waiting, minute by minute. Lets codee)
